opentelemetry/playbooks/FilelogRefusedLogs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: FilelogRefusedLogs
+weight: 20
+---
+
+# FilelogRefusedLogs
+
+## Problem
+
+## Impact
+
+## Diagnosis
+
+## Resolution Steps
```

Comment:
Great! We need to align on a common scheme how to provide this playbooks across Greenhouse. Put it on our to-do list.